cms/djangoapps/course_creators/migrations/0002_auto_20210418_0806.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 2.2.20 on 2021-04-18 08:06
```

Comment:
@farhaanbukhsh, nit: please use a proper name for the migration file instead of the auto-generated one.